src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml

```diff
@@ -0,0 +1,181 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.PriorLearningSummaryViewModel
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using SFA.DAS.ProviderCommitments.Features;
+@using SFA.DAS.ProviderCommitments.Web.RouteValues;
+@inject SFA.DAS.Authorization.Services.IAuthorizationService AuthorizationService
+
+@{
+    ViewData["Title"] = "Check your recognised prior learning";
+    ViewBag.ValidationOrder = "";
```

Comment:
As above, there are no GA tags

Reply:
Spoke to @ramnatarajanuk-dfe about this and because we are deprecating GA. He suggested we do not need to do this

Reply:
Ignore this previous comment. Spoke to Ram about this and we will add the GA tags